Tests/ApplicativeRouterTests/SyntaxRouterTests.swift

```diff
@@ -148,9 +148,16 @@ class SyntaxRouterTests: XCTestCase {
 
     XCTAssertEqual(route, router.match(request: request))
     XCTAssertEqual(request, router.request(for: route))
-    XCTAssertEqual(
-      "subscribe?plan=:__nscfnumber", // FIXME
-      router.templateUrl(for: route)?.absoluteString
-    )
+    #if os(Linux)
+      XCTAssertEqual(
+        "subscribe?plan=:int",
+        router.templateUrl(for: route)?.absoluteString
+      )
+    #else
+      XCTAssertEqual(
+        "subscribe?plan=:__nscfnumber", // FIXME
```

Comment:
Fun how this works in Linux but not macOS. Not sure if there's a way to coerce these `NSValue`s into their primitives. We may need to add a gnarly hack for pretty printing.